modules/m_cap.c

```diff
@@ -302,16 +302,11 @@ cap_ls(struct Client *source_p, const char *arg)
 static void
 cap_req(struct Client *source_p, const char *arg)
 {
-	static char buf_prefix[DATALEN + 1];
-	static char buf_list[2][DATALEN + 1];
-	const char *str_cont = " * :";
-	const char *str_final = " :";
-	int len_prefix;
-	int max_list;
+	char ack_buf[BUFSIZE];
```

Comment:
Let's make this `char ack_buf[DATALEN+1]` if that's what we need it to be.

Reply:
```suggestion
	char ack_buf[DATALEN+1];
```